packages/components/TestimonialSection.tsx

```diff
@@ -29,10 +34,12 @@ export default function TestimonialSection() {
         my="2rem"
         justifyItems="center"
       >
-        <div>Place Holder</div>
-        <div>Place Holder</div>
-        <div>Place Holder</div>
-        <div>Place Holder</div>
+        {testimonials.map((t) => (
+          <Box key={t.id!}>
+            <Text as="h2">{t.name}</Text>
+            <Text>{t.text}</Text>
+          </Box>
+        ))}
```

Comment:
```suggestion
        {testimonials.map(({ id, name, text }) => (
          <Box key={id!}>
            <Text as="h2">{name}</Text>
            <Text>{text}</Text>
          </Box>
        ))}
```

Acho queue seria melhor fazer um destructuring pra ficar mais legivel.

Acho que isso sera mudado na hora de integrar com o PR #132 , correto? Se sim, não precisa mudar aqui, mas lembra de fazer no proximo